### ISSUE TITLE
Can't install tamer with unified-planning 0.4.2.

### ISSUE THREAD

**Issue by racinmat** [aiplan4eu/unified-planning#295]
**Describe the bug**
It fails to install tamer on python 3.10.8 using Poetry, it throws error:
```
(aoc2022) PS C:\Projects\advent_of_code_2022> poetry add unified-planning[pyperplan,tamer]
Using version ^0.4.2 for unified-planning

Updating dependencies
Resolving dependencies...
Writing lock file

Package operations: 4 installs, 1 update, 0 removals

  • Updating wheel (0.37.1 -> 0.38.4)
  • Installing pytamer (0.1.1)

  RuntimeError


  at ~\.conda\envs\aoc2022\lib\site-packages\poetry\installation\chooser.py:105 in choose_for
      101│
      103│
      104│         if not links:
    → 105│             raise RuntimeError(f"Unable to find installation candidates for {package}")
      107│         # Get the best link
      108│         chosen = max(links, key=lambda link: self._sort_key(package, link))
      109│
```

When I install it using pip, it downgrades the unified planning to 0.2.2
```
(aoc2022) PS C:\Projects\advent_of_code_2022> pip install unified-planning[tamer]
Requirement already satisfied: unified-planning[tamer] in c:\users\azathoth\.conda\envs\aoc2022\lib\site-packages (0.4.2)
Requirement already satisfied: networkx in c:\users\azathoth\.conda\envs\aoc2022\lib\site-packages (from unified-planning[tamer]) (2.8.8)
Requirement already satisfied: pyparsing in c:\users\azathoth\.conda\envs\aoc2022\lib\site-packages (from unified-planning[tamer]) (3.0.9)
Collecting up-tamer==0.2.0
  Downloading up_tamer-0.2.0-py3-none-any.whl (12 kB)
INFO: pip is looking at multiple versions of unified-planning[tamer] to determine which version is compatible with other requirements. This could take a while.
Collecting unified-planning[tamer]
  Downloading unified_planning-0.4.2-py3-none-any.whl (230 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 230.4/230.4 kB 2.3 MB/s eta 0:00:00
  Downloading unified_planning-0.4.1-py3-none-any.whl (230 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 230.4/230.4 kB 14.7 MB/s eta 0:00:00
  Downloading unified_planning-0.4.0-py3-none-any.whl (230 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 230.4/230.4 kB 13.8 MB/s eta 0:00:00
  Downloading unified_planning-0.3.0-py3-none-any.whl (199 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 199.1/199.1 kB ? eta 0:00:00
Collecting up-tamer==0.1.0
  Downloading up_tamer-0.1.0-py3-none-any.whl (12 kB)
Collecting unified-planning[tamer]
  Downloading unified_planning-0.2.2-py3-none-any.whl (154 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 154.4/154.4 kB ? eta 0:00:00
WARNING: unified-planning 0.2.2 does not provide the extra 'tamer'
Installing collected packages: unified-planning
  Attempting uninstall: unified-planning
    Found existing installation: unified-planning 0.4.2
    Uninstalling unified-planning-0.4.2:
      Successfully uninstalled unified-planning-0.4.2
Successfully installed unified-planning-0.2.2
```

**To Reproduce**
1. Run `poetry add unified-planning[tamer]`

**Expected behavior**
It's all installed without error.

**Comment by alvalentini** [comment]
@racinmat thanks for the bug report.

In the last release, tamer is available also for python 3.10 on windows.

Can you check and, if solved, close the issue?

**Comment by racinmat** [comment]
Checked, it works now, thanks.
```
(aoc2022) PS C:\Projects\advent_of_code_2022> poetry add unified-planning[enhsp,fast-downward,pyperplan,tarski,tamer]
Using version ^0.5.0 for unified-planning

Updating dependencies
Resolving dependencies...

Writing lock file

Package operations: 11 installs, 0 updates, 0 removals

  • Installing wheel (0.38.4)
  • Installing antlr4-python3-runtime (4.7.2)
  • Installing multipledispatch (0.6.0)
  • Installing pyperplan (2.1)
  • Installing pytamer (0.1.10)
  • Installing tarski (0.8.2)
  • Installing up-fast-downward (0.0.6)
  • Installing up-enhsp (0.0.9)
  • Installing up-tamer (0.3.0)
  • Installing up-pyperplan (0.3.0)
  • Installing unified-planning (0.5.0)

```
